### ISSUE TITLE
Error loading when select a pod inside namespace

### ISSUE THREAD

**Issue by dnegrisolli** [Azure/vscode-kubernetes-tools#873]
Hello!
- Using Windows 10 19042.746, with VS Code in WSL 2
In this new version v.1.2.3 i´m facing some troubles with:
1- When I try to use the namespace and try to portfoward, appears the error:

```
Error loading document: Error: cannot open k8smsx://loadkubernetescore/pod-monitoring%20%20%20%20%20%20%20grafana-5c6f79684f-p5f2g.yaml?ns%3Dall%26value%3Dpod%2Fmonitoring%20%20%20%20%20%20%20grafana-5c6f79684f-p5f2g%26_%3D1612186166678. Detail: Unable to read file 'k8smsx://loadkubernetescore/pod-monitoring       grafana-5c6f79684f-p5f2g.yaml?ns=all&value=pod/monitoring       grafana-5c6f79684f-p5f2g&_=1612186166678' (Get command failed: error: there is no need to specify a resource type as a separate argument when passing arguments in resource/name form (e.g. 'kubectl get resource/<resource_name>' instead of 'kubectl get resource resource/<resource_name>'
)
```

2- when I trey to open a Pod in a Terminal:

`Failed to get containers in resource: error: there is no need to specify a resource type as a separate argument when passing arguments in resource/name form (e.g. 'kubectl get resource/<resource_name>' instead of 'kubectl get resource resource/<resource_name>'`

Thank you!

**Comment by lstocchi** [comment]
Hello @dnegrisolli, your issue looks similar to https://github.com/Azure/vscode-kubernetes-tools/issues/867 and it was fixed by https://github.com/Azure/vscode-kubernetes-tools/pull/868. You should see it working again from next release.

**Comment by dnegrisolli** [comment]
> Hello @dnegrisolli, your issue looks similar to #867 and it was fixed by #868. You should see it working again from next release.

I´m sorry! As I received a e-mail, i though it was the latest version.
